src/utils.js

```diff
@@ -0,0 +1,5 @@
+export const makeClassNameFromPath = (path) => path.split(process.platform === 'win32' ? "\\" : "/")
```

Comment:
using node's `path.sep` instead of detecting the separator slash by platform seems more stable

Reply:
wow
missed that path module member (it has been here for ages)

Reply:
fixed that